performance/sqlserver/backup_maintenance_planning.rst

```diff
@@ -0,0 +1,8 @@
+Backup maintenance plan
```

Comment:
Let's capitalize each word in this title to keep it in order with the other titles in the documentation.

Reply:
Good catch.